prow/config/config_test.go

```diff
@@ -26,3 +27,17 @@ func TestConfigLoads(t *testing.T) {
 		t.Fatalf("Could not load config: %v", err)
 	}
 }
+
+func TestConfigSecurityJobsMatch(t *testing.T) {
+	c, err := Load("../config.yaml")
+	if err != nil {
+		t.Fatalf("Could not load config: %v", err)
+	}
+	kp := c.Presubmits["kubernetes/kubernetes"]
+	sp := c.Presubmits["kubernetes-security/kubernetes"]
+	for i, j := range kp {
+		if strings.Replace(j.Name, "pull-kubernetes", "pull-security-kubernetes", -1) != sp[i].Name {
+			t.Fatalf("%s should match %s", j.Name, sp[i].Name)
+		}
```

Comment:
Deep Equals isn't going to work here, I tried and because of the nested-ness and pointers it won't work, I can go through all the args and match on them?